src/pacmod_node.cpp

```diff
@@ -38,7 +40,7 @@ using namespace AS::Drivers::PACMod;  // NOLINT
 
 double last_global_rpt_msg_received = 0.0;
 const double watchdog_timeout = 0.3;
-std::string veh_type_string = "POLARIS_GEM";
+std::string veh_type_string = "POLARIS_GEM";  // NOLINT
```

Comment:
Do we need provide the default value for vev_type_string? Can it be empty string?

Reply:
Could be empty, but currently the driver sets all other defaults here, so probably best to leave them together.